lib/keystore.py

```diff
@@ -684,11 +684,10 @@ def is_private_key_list(text):
 is_bip32_key = lambda x: is_xprv(x) or is_xpub(x)
 
 
-def bip44_derivation(account_id):
-    if bitcoin.TESTNET:
-        return "m/44'/1'/%d'"% int(account_id)
-    else:
-        return "m/44'/0'/%d'"% int(account_id)
+def bip44_derivation(account_id, segwit=False):
+    bip  = 49 if segwit else 44
```

Comment:
`purpose` and `coin_type` and `account` would mimic the BIP-0044 wording, but it's up to you.